jenkinsbox.yml

```diff
@@ -113,6 +94,21 @@
       - mysql
       - apache
 
+  roles:
```

Comment:
No need to move roles in between pre_tasks and tasks. Let's keep them in an old place